experimenter/experimenter/nimbus_ui_new/tests/test_forms.py

```diff
@@ -378,6 +378,31 @@ def test_valid_form_saves(self):
         )
         self.assertEqual(documentation_link.link, "https://www.example.com")
 
+    def test_name_field_is_required(self):
+        project = ProjectFactory.create()
+        documentation_link = NimbusDocumentationLinkFactory.create()
+
+        form_data = {
+            "hypothesis": "new hypothesis",
+            "risk_brand": True,
+            "risk_message": True,
+            "projects": [project.id],
+            "public_description": "new description",
+            "risk_revenue": True,
+            "risk_partner_related": True,
+            "documentation_links-TOTAL_FORMS": "1",
+            "documentation_links-INITIAL_FORMS": "1",
+            "documentation_links-0-id": documentation_link.id,
+            "documentation_links-0-title": (
+                NimbusExperiment.DocumentationLink.DESIGN_DOC.value
+            ),
+            "documentation_links-0-link": "https://www.example.com",
+        }
+
+        form = OverviewForm(data=form_data)
+
+        self.assertFalse(form.is_valid())
```

Comment:
Cool, couple things here:

Instead of removing the 'name' key from the data altogether, try having it and setting it to an empty string. 

Also should explicitly check that the 'name' field is in`form.errors`